Resources/Prototypes/Catalog/VendingMachines/Inventories/chemdrobe.yml

```
@@ -5,11 +5,17 @@
     ClothingUniformJumpskirtChemistry: 2
     ClothingOuterCoatLabChem: 2
     ClothingShoesColorWhite: 2
+    ClothingUniformJumpsuitChemShirt: 2 #Delta V: Add a new chemist jumpsuit, formal variant
+    ClothingNeckTieChem: 2  #Delta V: Add chemist's tie to go along a new variant of their suit
+    ClothingShoesEnclosedChem: 2 #Delta V: Add realistic chemistry shoes
     ClothingBackpackChemistry: 2
+    ClothingOuterApronChemist: 2 #Delta V: Add realistic PPE apron for chemistry
     ClothingBackpackSatchelChemistry: 2
     ClothingBackpackDuffelChemistry: 2
     ChemBag: 2
     ClothingBeltMedical: 2
+    ClothingEyesGlassesChemist: 2 #Delta V: Add safety glasses
+    ClothingHandsGlovesChemist: 2 #Delta V: Add heavy duty chemistry gloves
     ClothingHandsGlovesLatex: 2
     ClothingHeadsetMedical: 2
     ClothingOuterWinterChem: 2
```

```suggestion
    ClothingBackpackChemistry: 2
    ClothingBackpackSatchelChemistry: 2
    ClothingBackpackDuffelChemistry: 2
    ChemBag: 2
    ClothingBeltMedical: 2
    ClothingHandsGlovesLatex: 2
    ClothingHeadsetMedical: 2
    ClothingOuterWinterChem: 2
    # DeltaV - Personal Protective Equipment - Add various PPE equipment including new clothings
    ClothingUniformJumpsuitChemShirt: 2 # New chemist jumpsuit, formal variant
    ClothingNeckTieChem: 2  # Chemist's tie to go along a new variant of their suit
    ClothingShoesEnclosedChem: 2 # Realistic chemistry shoes
    ClothingOuterApronChemist: 2 # Realistic PPE apron for chemistry
    ClothingEyesGlassesChemist: 2 # Safety glasses
    ClothingHandsGlovesChemist: 2 # Heavy duty chemistry gloves
    # End of modified code
```
Changes marker with new marker, see https://github.com/DeltaV-Station/Delta-v/blob/master/MARKERS.md#yaml
Also batches all changes together.